### PR TITLE
Update docs with tokens

### PR DESCRIPTION
**Pull request by genestack-solomatin**

### CODE REVIEW

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -58,7 +58,7 @@ To avoid typing in your credentials every time you connect to a Genestack instan
 
     $ genestack-user-setup init
 
-You will be prompted for your email and password to connect to Genestack. If they are valid and the connection to the Genestack server is successful, you're all set!
+You will be prompted for your email and password/token to connect to Genestack. If they are valid and the connection to the Genestack server is successful, you're all set!
```
Comment (klas-genestack) [inline docs/source/getting_started.rst]: Make that "token or email and password", would be much more clear.

File: docs/source/getting_started.rst [hunk 2]
```diff
@@ -71,13 +71,13 @@ To check the result, you can run:
 
 .. warning::
 
-   By default, your password will be stored using a secure storage system provided by your OS (See https://pypi.python.org/pypi/keyring for more information)
+   By default, your password/token will be stored using a secure storage system provided by your OS (See https://pypi.python.org/pypi/keyring for more information)
```
Comment (klas-genestack) [inline docs/source/getting_started.rst]: "your passwords and/or tokens"

Comment (klas-genestack) [reply docs/source/getting_started.rst]: Also "see" in parenthesis shouldn't be capitalized.

File: docs/source/getting_started.rst [hunk 3]
```diff
@@ -71,13 +71,13 @@ To check the result, you can run:
...
    If the secure storage system is not accessible, you will be asked for permission to store your password in plain text in a configuration file. However, this option is **strongly discouraged**. You have been warned!
 
 
 .. note::
 
-   The information you supply to ``genestack-user-setup`` is only stored locally on your computer. Therefore, if you change your password online, you will need to update your local configuration as well.
+   The information you supply to ``genestack-user-setup`` is only stored locally on your computer. Therefore, if you change your password/token online, you will need to update your local configuration as well.
```
Comment (klas-genestack) [inline docs/source/getting_started.rst]: "if you change your password or token in Platform UI"

File: docs/source/getting_started.rst [hunk 4]
```diff
@@ -132,7 +132,7 @@ Using the connection parameters, you can run this script from a terminal using d
 .. TODO talk more about the parser and how you shouldn't use get_connection()
 
 If your script accepts custom command-line arguments, you can add them to the arguments parser returned by :py:func:`~genestack.make_connection_parser`.
-The arguments ``-u``, ``-p`` and ``-H`` are reserved for the connection parameters.
+The arguments ``-u``, ``-p``, ``-H``, ``--host``, ``--api-token``, ``--show-logs`` and ``--debug`` are reserved for the connection parameters.
```
Comment (klas-genestack) [inline docs/source/getting_started.rst]: Looks weird to me.  I think the list should look like `--full-option` (`-o`): of course, if there is short option.

File: docs/source/getting_started.rst [hunk 5]
```diff
@@ -170,10 +170,12 @@ If no connection parameter is passed to your script, ``get_connection`` will att
 If only the parameter ``-u <alias>`` is supplied, the parser will look for the corresponding identity in the local configuration file. If no match is found, the script will switch to interactive login.
 
 You can also supply the parameters ``-u <email> -H <host> -p <password>``. By default, the host is ``platform.genestack.com`` and if no password is provided, you will be prompted for one.
+Or you can supply ``-H <host> --token <token>`.
```
Comment (klas-genestack) [inline docs/source/getting_started.rst]: `--token` like here, or `--api-token` as above?

Comment (genestack-solomatin) [reply docs/source/getting_started.rst]: `--token`  is correct one

**Pull request merged**